core/src/test/scala/unit/kafka/admin/TopicCommandTest.scala

```diff
@@ -16,585 +16,41 @@
  */
```

Comment:
This file was actually moved/renamed to core/src/test/scala/unit/kafka/admin/TopicCommandWithZKClientTest.scala